packages/ui/src/theme/tokens/components/copy.ts

```diff
@@ -1,6 +1,6 @@
 export const copy = {
-  fontSize: { value: '{font.sizes.xs.value}' },
-  gap: { value: '{space.relative.medium.value}' },
+  fontSize: { value: '{fontSizes.xs}' },
```

Comment:
Why `.value` could be skipped here? It seems we always need that when referencing other tokens value

Reply:
Correct, `.value` can be skipped now

Reply:
+1 to this question, thought I was just missing something

Reply:
Do we need to do this on all the other tokens too, or is it an optional change?

Reply:
It is optional, we could do a PR to remove it for all the other tokens because it is cleaner